genome_designer/utils/data_export_util.py

```diff
@@ -181,7 +185,8 @@ def export_var_dict_list_as_vcf(var_dict_list, vcf_dest_path_or_filehandle,
                 1, # QUAL
                 [], # FILTER
                 {
-                    'METHOD': 'DE_NOVO_ASSEMBLY'
+                    'METHOD': 'DE_NOVO_ASSEMBLY',
+                    'CALLER': caller_string
```

Comment:
I like having this information, but I think the key `CALLER` is a bit ambiguous and hard to distinguish from `METHOD`. Also, we use the term "caller" to refer to tools like Freebayes and Lumpy. And we use the METHOD key to set which "caller" was used (e.g. LUMPY).

This new field refers to the sub-method used for a given caller? Would a better key be `SUBMETHOD`?